[CUBRIDMAN-192] Remove oracleStyleNumberReturn property of JDBC/CCI (#456)
CUBRID/cubrid-manual: en/admin/config.rst

        ==========
               0.5

    .. note:: 

>         If oracle_compat_number_behavior is set to yes, the data must be read in String format to prevent JDBC/CCI from outputting decimal point trailing zeros for NUMERIC, DOUBLE, and FLOAT types. Please refer to the usage instructions below.

Reviewer: If the oracle_compat_number_behavior system parameter is set to yes, the data must be read in String format to prevent JDBC/CCI from outputting trailing zeros after decimal point for NUMERIC, DOUBLE, and FLOAT types. Please refer to the usage instructions

Author: I applied mhoh3963's suggestion.

>         If oracle_compat_number_behavior is set to yes, the data must be read in String format to prevent JDBC/CCI from outputting decimal point trailing zeros for NUMERIC, DOUBLE, and FLOAT types. Please refer to the usage instructions below.

Reviewer 2: ```suggestion
        The oracle_compat_number_behavior is only applied, when reading NUMERIC, DOUBLE, and FLOAT type data in string format in JDBC/CCI.  The related functions of JDBC/CCI are as follows.
```

Author: fixed it.

>         * In JDBC, you must use the getString(int columnIndex) or getString(String columnLabel) and getObject(int columnIndex) or getObject(String columnLabel) methods.

Reviewer: * In JDBC, use the getString(int columnIndex) or getString(String columnLabel) and getObject(int columnIndex) or getObject(String columnLabel) methods.

Author: I applied mhoh3963's suggestion.

>         * In JDBC, you must use the getString(int columnIndex) or getString(String columnLabel) and getObject(int columnIndex) or getObject(String columnLabel) methods.

Reviewer 2: ```suggestion
        * JDBC : getString(int columnIndex), getString(String columnLabel), getObject(int columnIndex) , getObject(String columnLabel)
```

Author: fixed it.

>         * In CCI, CCI_A_TYPE_STR must be used in cci_get_data(...). Example) cci_get_data(req, i, CCI_A_TYPE_STR, &data, &ind);

Reviewer: * In CCI, use CCI_A_TYPE_STR as the type when calling cci_get_data (...), for example, cci_get_data(req, i, CCI_A_TYPE_STR, &data, &ind);

Author: I applied mhoh3963's suggestion.

>         * In CCI, CCI_A_TYPE_STR must be used in cci_get_data(...). Example) cci_get_data(req, i, CCI_A_TYPE_STR, &data, &ind);

Reviewer 2: ```suggestion
        * CCI :  cci_get_data (CCI_A_TYPE_STR as type), Example) cci_get_data(req, i, CCI_A_TYPE_STR, &data, &ind)
```

Author: fixed it.
